user_guides/jakartaee/src/main/jbake/content/intro.adoc

```diff
@@ -140,77 +140,122 @@ harness.
 
 [[sthref4]][[jakarta-ee-10-technologies-tested-with-jakarta-ee-10-platform-tck]]
 
-1.2.1 Jakarta EE 10 Technologies Tested with Jakarta EE 10 Platform TCK
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-The Jakarta EE 10 Platform TCK test suite includes compatibility tests for the
-following required and optional Jakarta EE 10 technologies:
-
-* Jakarta Authentication 3.0
-* Jakarta Authorization 2.1
-* Jakarta Bean Validation 3.0
-* Jakarta Common Annotations 2.1
-* Jakarta Contexts and Dependency Injection 4.0
-* Jakarta Concurrency 3.0
-* Jakarta Connectors 2.1
-* Jakarta Debugging Support for Other Languages 2.0
-* Jakarta Dependency Injection 2.0
-* Jakarta Enterprise Beans 4.0
-* Jakarta Expression Language 5.0
-* Jakarta Faces 4.0
-* Jakarta JSON Processing 2.1
-* Jakarta JSON Binding 3.0
-* Jakarta Interceptors 2.1
-* Jakarta Mail 2.1
-* Jakarta Messaging 3.1
-* Jakarta Persistence 3.1
-* Jakarta RESTful Web Services 3.1
-* Jakarta Security 3.0
-* Jakarta Server Pages 3.1
-* Jakarta Servlet 6.0
-* Jakarta Standard Tag Library 3.0
-* Jakarta Transactions 2.0
-* Jakarta WebSocket 2.1
-* Jakarta Enterprise Beans 3.2 and earlier entity beans and associated Jakarta Enterprise Beans QL (optional)
-* Jakarta Enterprise Beans 2.x API group (optional)
-* Jakarta Enterprise Web Services 4.0 (optional)
-* Jakarta SOAP with Attachments 3.0 (optional)
-* Jakarta XML Web Services 3.0 (optional)
-
+1.2.1 Jakarta EE 10 Technologies Required for Jakarta EE 10 Platform Compatibility
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The Jakarta EE 10 Platform Specification defines the required and optional component specifications.
+The full list with specification version requirements is defined in the Platform EE Specification document (https://jakarta.ee/specifications/platform/10/), see the heading 'Full Jakarta™ EE Product Requirements,'
+
+Jakarta EE 10 TCK tests verify partial compatibility for the Jakarta EE Platform.
+The Platform TCK includes tests for the following components:
+
+* Jakarta Activation
+* Jakarta Authentication
+* Jakarta Authorization
+* Jakarta Batch
+* Jakarta Bean Validation
+* Jakarta Common Annotations
+* Jakarta Concurrency
+* Jakarta Connectors
+* Jakarta Contexts and Dependency Injection
+* Jakarta Debugging Support for Other Languages
+* Jakarta Dependency Injection
+* Jakarta Enterprise Beans (also, see optional below)
+* Jakarta Enterprise Web Services
+* Jakarta Expression Language
+* Jakarta Server Faces
+* Jakarta Interceptors
+* Jakarta JSON Binding
+* Jakarta JSON Processing
+* Jakarta Mail
+* Jakarta Messaging
+* Jakarta Persistence
+* Jakarta RESTful Web Services
+* Jakarta Security
+* Jakarta Server Pages
+* Jakarta Servlet
+* Jakarta Standard Tag Library
+* Jakarta Transactions
+* Jakarta WebSocket
+
+The following optional technologies may also tested via the Jakarta EE Platform TCK:
```

Comment:
minor: change from `may also tested` to `may also be tested`.

Reply:
Fixed. Thank you.